Description: fix Gruntfile
Author: Xavier Guimard <yadd@debian.org>
Forwarded: not-needed
Last-Update: 2020-02-21

--- a/Gruntfile.js
+++ b/Gruntfile.js
@@ -43,9 +43,9 @@
   grunt.loadTasks('tasks');
 
   // These plugins provide necessary tasks.
-  grunt.loadNpmTasks('grunt-contrib-jshint');
+  //grunt.loadNpmTasks('grunt-contrib-jshint');
   grunt.loadNpmTasks('grunt-contrib-nodeunit');
-  grunt.loadNpmTasks('grunt-contrib-internal');
+  //grunt.loadNpmTasks('grunt-contrib-internal');
 
   // Setup a test helper to create some folders to clean.
   grunt.registerTask('copy', 'Copy fixtures to a temp location.', function() {
@@ -69,8 +69,8 @@
 
   // Whenever the 'test' task is run, first create some files to be cleaned,
   // then run this plugin's task(s), then test the result.
-  grunt.registerTask('test', ['jshint', 'copy', 'clean', 'nodeunit']);
+  grunt.registerTask('test', ['copy', 'clean', 'nodeunit']);
 
   // By default, lint and run all tests.
-  grunt.registerTask('default', ['jshint', 'test', 'build-contrib']);
+  grunt.registerTask('default', ['test', 'build-contrib']);
 };
